go.mod

```diff
@@ -82,13 +82,11 @@ require (
 	github.com/go-openapi/jsonreference v0.19.3 // indirect
 	github.com/go-openapi/spec v0.19.3 // indirect
 	github.com/go-openapi/swag v0.19.5 // indirect
-	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0 // indirect
```

Comment:
Some of these were added manually and need to stay because of how we install ginkgo for certain actions. See https://github.com/aws/amazon-vpc-cni-k8s/pull/2112 , which links to the action workflow that found this

Reply:
Added the slim-sprig and pprof packages back.